### PR TITLE
Removed contacts page and added footer to remaining pages

### PR DESCRIPTION
**Pull request by conste11ations**
Down the line, I think it would be best to create subfolders for sub-components under components.
For example, Angular style guide recommends nesting when number of files per level (of the hierarchy) >= 7 files.

Screen Shot 2020-12-12 at 8 10 03 PM

### CODE REVIEW

File: src/components/footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                    <Icon icon={'envelope'} url={'mailto:wwgtoronto@gmail.com'}></Icon>
+                    <Icon icon={'slack'} url={'#'}></Icon>
+                    <Icon icon={'github'} url={'https://github.com/WWGToronto'}></Icon>
+                </div>
+            </div>
+            <div className="mt4">
+                <a href="/" className="f6 link dim gray dib mr3 mr4-ns">Home</a>
+                <a href="/events" className="f6 link dim gray dib mr3 mr4-ns">Events</a>
+                <a href="/sponsors" className="f6 link dim gray dib mr3 mr4-ns">Sponsors</a>
+                <a href="#" className="f6 link dim gray dib">Code of Conduct</a>
```
Comment (rayhaneh) [inline src/components/footer/index.tsx:20]: Where is this going to link to? CoC on meetup or a new page?

File: src/layouts/sponsors.tsx [hunk 2]
```diff
@@ -3,14 +3,14 @@ import * as React from 'react';
...
+  Footer,
   SponsorInfo,
 } from '../components';
 
-
 export class Sponsors extends React.Component {
   public render(): JSX.Element {
     return (
-      <div className={'pt5'}>
+      <>
```
Comment (rayhaneh) [inline src/layouts/sponsors.tsx:13]: What does an empty html element mean if you don't mind me asking?

Comment (conste11ations) [reply src/layouts/sponsors.tsx:13]: They are the short syntax for [React fragments](https://reactjs.org/docs/fragments.html#short-syntax).  You can return multiple elements without needing to put an extra `<div>` in the DOM at the top-level of your return statement.

Comment (rayhaneh) [reply src/layouts/sponsors.tsx:13]: Pretty cool!

File: src/components/footer/index.tsx (hunk 1 above)
Comment (conste11ations) [reply src/components/footer/index.tsx:20]: At the time I was thinking of linking to our CoC on meetup as a short-term solution. In the long-term, perhaps the Code of Conduct would be housed here on our website, for being a member of our club in general. Then we can put the link on event promotion websites not limited to meetup (eventbrite, etc).

**Review by rayhaneh — APPROVED** [review]
Thanks Rachel! Also feel free to create the subfolders you mentioned if you have the time. Also let's chat about that CoC on Monday :)

**Comment by conste11ations** [comment]
> Thanks Rachel! Also feel free to create the subfolders you mentioned if you have the time. Also let's chat about that CoC on Monday :)

Will do! And yes we can discuss this Monday :) Have a good weekend!

**Pull request merged**